.python-version

```diff
@@ -0,0 +1 @@
+3.12
```

Comment:
Too bad it's not in the pyproject.toml 😢 
Can it use more than one version? Restricting to 3.12 might prevent people from using prx